pkg/systemd/services/service-details.jsx

```diff
@@ -601,81 +622,74 @@
-                                        <Tooltip id="switch-unit-state" content={tooltipMessage} position={TooltipPosition.right}>
-                                            <span>
-                                                <Switch isChecked={enabled}
-                                                        aria-label={tooltipMessage}
-                                                        isDisabled={this.state.waitsAction || this.state.waitsFileAction}
-                                                        onChange={this.onOnOffSwitch} />
-                                            </span>
-                                        </Tooltip>
-                                    }
-                                    <ServiceActions { ...{ active, failed, enabled, masked } } canReload={this.props.unit.CanReload}
-                                                    actionCallback={this.unitAction} fileActionCallback={this.unitFileAction}
-                                                    deleteActionCallback={isCustom && isTimer ? this.deleteAction : null}
-                                                    disabled={this.state.waitsAction || this.state.waitsFileAction}
-                                                    isPinned={this.state.isPinned} pinUnitCallback={this.pinUnit} />
-                                </>
+                <CardTitle className="service-top-panel">
+                    <Text component={TextVariants.h2} className="service-name">{this.props.unit.Description}</Text>
+                    {this.state.isPinned &&
+                        <Tooltip content={_("Pinned unit")}>
+                            <ThumbtackIcon className='service-thumbtack-icon' />
+                        </Tooltip>}
+                    { showAction &&
+                        <>
+                            { !masked && !isStatic &&
+                                <Tooltip id="switch-unit-state" content={tooltipMessage} position={TooltipPosition.right}>
+                                    <span>
+                                        <Switch isChecked={enabled}
+                                                aria-label={tooltipMessage}
+                                                isDisabled={this.state.waitsAction || this.state.waitsFileAction}
+                                                onChange={this.onOnOffSwitch} />
+                                    </span>
+                                </Tooltip>
                             }
-                        </CardTitle>
-                        <CardBody>
+                            <ServiceActions { ...{ active, failed, enabled, masked } } canReload={this.props.unit.CanReload}
+                                            actionCallback={this.unitAction} fileActionCallback={this.unitFileAction}
+                                            deleteActionCallback={isCustom && isTimer ? this.deleteAction : null}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18422-20230308-112739-0cbbb576-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)